Code/BatteryGate.cs

```diff
@@ -52,7 +52,7 @@ public BatteryGate(Vector2 position, int size, bool vertical, int? openWith, boo
         }
 
         public BatteryGate(EntityData data, Vector2 offset, EntityID id)
-            : this(data.Position + offset, data.Height, data.Bool("vertical"), data.Int("switchId", -1), data.Bool("closes", false), id) {
+            : this(data.Position + offset, data.Height != 0 ? data.Height : 48, data.Bool("vertical"), data.Int("switchId", -1), data.Bool("closes", false), id) {
```

Comment:
Yeah, basically the Ahorn plugin sets height to 48, but it doesn't show up as a field and there's no way to change it (probably because the placements use `point`), so I have no idea why the field exists in the first place. But if I add height in Lönn, I can't hide it with ignoredFields, presumably because of special handling (I might need to report that again, I think it got drowned in all the convos last night), and changing it will break stuff (at least if greater than 48, I need to check if smaller values actually work). So I'm just not setting height in Lönn at all.

Reply:
Checked with PenguinOwl about the height stuff, she's looking into it.

Reply:
Hardcoded height code side now since it was never modifiable in Ahorn and just causes issues with Lönn.